components/MoveList.js

```diff
@@ -18,8 +20,11 @@ export default class MoveList extends React.Component {
         key={JSON.stringify(movelist)}/>
     ));
 
+    const renderedMechanics = !!mechanics && <Mechanics mechanicsTitle="Mechanics" mechanics={mechanics}/>
+
     return (
       <List>
+        {renderedMechanics}
```

Comment:
Would it make more sense to have the mechanics section after the moves? To me that kinda seems more a better flow?

Reply:
I was thinking about that. Obviously having it at the top would make it more noticeable but I guess the question is whether that would just get in the way.

Being at the bottom would probably emphasize the nature as a reference tool rather than a learning tool, so I think it makes sense.

Reply:
:+1: for ref tool